car-search/backend/CarSearchAPI/CarSearchAPI/Abstractions/IOfferPageSeparator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CarSearchAPI.DTOs.CarSearch;
+using CarSearchAPI.DTOs.ForwardingParameters;
+
+namespace CarSearchAPI.Abstractions
+{
+    public interface IOfferPageSeparator
```

Comment:
I think this would be better off renamed to IOfferPageService. An implementation is called similarly, and that "Separator" part just doesn't sit right with me as this does more to pages than just separating them